modules/file-rotation/file-rotation.c

```diff
@@ -63,6 +87,26 @@ file_rotation_set_size(FileRotationPlugin *self, gsize size)
   self->size = size;
 }
 
+void
+file_rotation_set_date_format(FileRotationPlugin *self, gchar *date_format)
+{
+  if (date_format[0] != '-')
```

Comment:
I think we should let the user specify any format they want.

Reply:
Removing the conditions then